coverage/src/main/scala/com/databricks/labs/remorph/coverage/AcceptanceTestSource.scala

```diff
@@ -10,6 +11,22 @@ trait AcceptanceTestSource {
   def listTests: Seq[AcceptanceTest]
 }
 
+object NestedFiles {
+  def projectRoot: String = checkProjectRoot(Paths.get(".")) match {
+    case p if p == Paths.get("/") => throw new RuntimeException("Could not find project root")
+    case p => p.toString
+  }
+
+  @tailrec private def checkProjectRoot(current: Path): Path = {
+    // check if labs.yml exists in the current folder
+    if (Files.exists(current.resolve("labs.yml"))) {
+      current
+    } else {
+      checkProjectRoot(current.getParent)
+    }
```

Comment:
Suggested version would work even in the (arguably very unlikely) case where `labs.yml` is located directly under `/`.

```suggestion
  def projectRoot: String = checkProjectRoot(Paths.get(".")).toString

  @tailrec private def checkProjectRoot(current: Path): Path = {
    // check if labs.yml exists in the current folder
    if (Files.exists(current.resolve("labs.yml"))) {
      current
    } else {
      if (current.getParent != null) {
        checkProjectRoot(current.getParent)
      } else {
        throw new RuntimeException("Could not find project root")
      }
    }
```